ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/converters/EventStreamHttpConverter.java

```diff
@@ -61,7 +61,7 @@ public void write(@NotNull EventStreamResponse eventStreamResponse, MediaType co
 		Model eventStreamModel = eventStreamResponseConverter.toModel(eventStreamResponse);
 		Lang lang = rdfModelConverter.getLang(contentType, REST_ADMIN);
 		rdfModelConverter.checkLangForRelativeUrl(lang);
-
+		outputMessage.getHeaders().setContentType(MediaType.valueOf(lang.getHeaderString()));
```

Comment:
Can't you use the 'contentType' parameter that you receive here? Now you convert this param to a lang and then extract it from the lang and map it back to your MediaType that you started with..